packages/@aws-cdk/aws-eks/README.md

```diff
@@ -1144,6 +1144,24 @@ cluster.addHelmChart('test-chart', {
 });
 ```
 
+### OCI Charts
+
+OCI charts are also supported. Before executing ensure the handler lambda has the required ECR IAM permissions.
```

Comment:
What are the required permissions? Can we automate that for the user?

Reply:
They would be ecr repository permissions to be able to log in. Ideally, it
should be present on the lambda.

Reply:
The handler role is this one:

https://github.com/aws/aws-cdk/blob/19a287f02bc427644837956466213ee65457a857/packages/@aws-cdk/aws-eks/lib/kubectl-provider.ts#L164

You will need to add the `ecr:GetAuthorizationToken` permission to it

Reply:
@otaviomacedo , Done. please review

Reply:
Looks good. But the tests have to be updated now.

Reply:
@otaviomacedo , I tried to look for a test under cdk-eks. Cant see anything. I assumed there are no tests for the same. Neither the documentation tells how all of this is actually tested.

If you can point me to the location. I am happy to write tests.

Reply:
@otaviomacedo , When I am trying to run tests from aws-eks directory. It appears to be not picking up tests at all or giving other errors. Could you advise me on the correct steps to execute tests?

Reply:
`yarn build && yarn test` should be the only thing you need. Another thing you can try is to build the whole transitive closure before running the test:

```sh
 aws-eks $ ../../../scripts/buildup
```

Reply:
@otaviomacedo , the tests are now complete. Could we please merge this ?

Reply:
@, Could you please review. Thanks